source/guide_mailman-3.rst

```diff
@@ -464,6 +464,100 @@ Install cronjobs
+  set daemon 60
+  set no syslog
+  #has no effect with --nodetach, but ensures that we log to the same file, when we debug manually
+  set logfile /home/isabell/var/logs/fetchmail_out.log
+
+  #BEGIN/END are only for readability and not part of fetchmails free form syntax
+  #but this synatx is very prone to subtile errors, so a littly structure might help
+
+  poll stardust.uberspace.de
+    #BEGIN server options
+    port 993
+    protocol imap
+    #these are specific to Qmail, need to be changed, when the MDA changes
+    #alternatively both might be "emulatable" with sieve later
+    qvirtual isabell-
+    envelope Delivered-To
+    localdomains isabell.uber.space
+    #END of server options
+    #
+    #BEGIN user options
+    username isabell-catchalldefault
+    #for whatever reason 'ssl' is a user option and must come *after* username
+    ssl
+    idle
+    password ""
+    no fetchall
+    folder Mailman
+    #this specifies mailmans *L*MTP server with port 8024:
+    smtphost isabell.local.uberspace.de/8024
+    #but now we tell fetchmail, that this is actually LMTP:
+    lmtp
+    #deduce the receiver from the envelope
+    to *
+    #END user options
+
+This file still contains two options that are qmail specific. The option ``qvirtual isabell-`` tells fetchmail to strip all prefixes added by qmail and only use the remaining part as the recipient address. The ``envelope`` options tells fetchmail where to get the original envelope headers that where processed by the MDA. Eech MDA has different headers where they store this information and ``Delivered-To`` is qmail's way of passing the information. When Uberspace finally switches to some other MDA, you can drop the ``qvirtual`` option and alter ``Delivered-To``. Note that sieve filters can set addtionial headers based on the envelope, so you might also be able to emulate qmails beahvior with sieve later.
+
+Finally, we also need an supervisord unit file for fetchmail, which we will place at ``/home/isabell/etc/services.d/fetchmail.ini``:
+
+.. code:: ini
+
+  directory=%(ENV_HOME)s
+  command=/usr/bin/fetchmail --nodetach -vvv -f %(ENV_HOME)s/mailman-suite/fetchmailrc
+  autostart=true
+  autorestart=true
+  stderr_logfile=%(ENV_HOME)s/var/logs/fetchmail_err.log
+  stdout_logfile=%(ENV_HOME)s/var/logs/fetchmail_out.log
+  stopsignal=TERM
+  # `startsecs` is set by Uberspace monitoring team, to prevent a broken service from looping
+  startsecs=30
+
+Most of this is pretty much standard, but note the ``--nodetach`` option. This stops fetchmail from forking and exiting. As ``--nodetach`` disables fetchmail's logfile option, we let supervisord forward its output to the same file (except for errors). You can omit the ``-vvv`` verbosity option later, but it is very helpful for debugging fetchmail's deduction rules for adresses which can be quite surprising. Finally we start fetchmail:
+
+.. code-block:: bash
+
+  servicectl reread
```

Comment:
Shouldn't this be supervisorctl?

Reply:
Yes, sorry, `supervisorctl` is of course correct.

Reply:
Also I just noticed that there is a

`supervisorctl update
`

missing

Reply:
Oh, that's even more important. This explains, why I had to `reload`. I checked my history, `update` was indeed missing.

Reply:
OK